src/scripts/clipperUI/frontEndGlobals.ts

```diff
@@ -5,25 +5,55 @@ import {Communicator} from "../communicator/communicator";
 import {Logger} from "../logging/logger";
 import {Constants} from "../constants";
 
-export module Clipper {
-	export let injectCommunicator: Communicator;
-	export let extensionCommunicator: Communicator;
-	export let logger: Logger;
-	export let sessionId: SmartValue<string> = new SmartValue<string>();
+import {RemoteStorage} from "../storage/remoteStorage";
+import {StorageAsync} from "../storage/storageAsync";
 
-	export function getUserSessionId(): string {
-		return this.sessionId.get();
+export class Clipper {
+	private static injectCommunicator: Communicator;
+	private static extensionCommunicator: Communicator;
+
+	private static storage: StorageAsync;
+
+	// Public for convenience?
```

Comment:
Good idea. I left this as public static because this file used to be a module rather than a class. I wanted to keep the changes as minimal as possible so it's not a nightmare to review, so I wanted to keep the get/set syntax the same. I'll add an Issue.